extensions/oidc/runtime/src/main/java/io/quarkus/oidc/VertxJwtCallerPrincipal.java

```diff
@@ -9,21 +9,13 @@
  */
 public class VertxJwtCallerPrincipal extends DefaultJWTCallerPrincipal {
     private JwtClaims claims;
-    private String customPrincipalName;
```

Comment:
Note I've removed this code because it came from the smallrye-jwt copy/paste and it was not supposed to be there in the 1st place, the user name can only come from the JWT itself and smallrye-jwt checks various properties itself